canonical-web-and-design/www.ubuntu.com
# Remove _html5 shiv as no longer required

## Done

Remove _html5 shiv as none of our supported browsers require a HTML5 shiv

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Sanity check code
